<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendOrder</class>
 <widget class="QWidget" name="SendOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>下单</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>371</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1,1,1,2,1,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,4,2">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>账户:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="account"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="1,4,2">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>合约:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="instrument"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="1,4">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>买卖:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_5">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="1,1,1">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="buy">
           <property name="text">
            <string>买入</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="sell">
           <property name="text">
            <string>卖出</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="1,4">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>开平:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_6">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="open">
           <property name="text">
            <string>开仓</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="close">
           <property name="text">
            <string>平仓</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="closeToday">
           <property name="text">
            <string>平今</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_19" stretch="1,4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>价格:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame_7">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_20" stretch="2,3,1">
           <item>
            <widget class="QRadioButton" name="priceLimit">
             <property name="text">
              <string>限价</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="price"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_11">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_21" stretch="1,1,1">
           <item>
            <widget class="QRadioButton" name="priceMarket">
             <property name="text">
              <string>市价</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="priceRival">
             <property name="text">
              <string>对手价</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="priceNew">
             <property name="text">
              <string>最新价</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_22" stretch="1,2,2">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>数量:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="quantity"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_12">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_23" stretch="1,4">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>类型:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_8">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_24" stretch="1,1,1">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="Speculation">
           <property name="text">
            <string>投机</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="Arbitrage">
           <property name="text">
            <string>套利</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="Hedge">
           <property name="text">
            <string>保值</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
